react/components/SelectedFilters.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage, injectIntl } from 'react-intl'
+import { Collapse } from 'react-collapse'
+import { reduce, concat, map, keys } from 'ramda'
+
+import Checkbox from '@vtex/styleguide/lib/Checkbox'
```

Comment:
idem

Reply:
In this case is not possible, because there's not Checkbox component in the vtex.styleguide app